src/Renci.SshNet.Tests/Classes/SshCommandTest.cs

```diff
@@ -56,6 +56,70 @@ public void Test_Run_SingleCommand()
             }
         }
 
+#if FEATURE_TAP
+        [TestMethod]
+        [TestCategory("integration")]
+        public async Task Test_Run_SingleCommandAsync()
+        {
+            var host = Resources.HOST;
+            var username = Resources.USERNAME;
+            var password = Resources.PASSWORD;
+
+            using (var client = new SshClient(host, username, password))
+            {
+                #region Example SshCommand RunCommand Result Async
+                await client.ConnectAsync(default);
+
+                var testValue = Guid.NewGuid().ToString();
+                var command = await client.RunCommandAsync(string.Format("echo {0}", testValue));
+                var result = await new StreamReader(command.OutputStream).ReadToEndAsync();
```

Comment:
For commands with little output this is OK, but if this is to serve as a general example, I propose to use something like code below to indicate that the output can be processed parallelly while the command is executing:
```
            Task execTask = command.ExecuteAsync(cancellationTokenSource.Token);
//start a task to process output
            Task outputTask = ProcessOutputAsync(new StreamReader(command.OutputStream));
           await Task.WhenAll(execTask, outputTask);
```
Also, please always provide cancellationToken in the examples. I'll come back to this point later...